### PR TITLE
[ci] enable sival tests in CI

### PR DESCRIPTION
**Pull request by nbdd0121**

### CODE REVIEW

**Comment by jwnrt** [comment]
We can let this run through CI to get a good idea of how long the tests will add, but it might be a good idea to separate them into another job like we've done for manufacturing tests.

**Review by rswarbrick — APPROVED** [review]
How exciting! :-)

As a follow-up, I wonder whether it would make sense to stick these tests in a variable on the previous line. It might let us cut down on the slightly crazy long line that we're building here!

**Comment by nbdd0121** [comment]
I tried to run the test locally and it also times out. Increasing the timeout doesn't seem to work either. This might be a genuine problem. The test used to be functional, but after #20402 we stopped running it in CI at all, and it seems that the PR also breaks it. @moidx

**Comment by jwnrt** [comment]
I think:

* If the test can be fixed easily (by yourself or @moidx), then feel free to try fix it before this PR.
* Otherwise, please mark the test as broken and we can merge this to prevent other breakages like it.

**Review by jwnrt — APPROVED** [review]
LGTM, thanks @nbdd0121

File: azure-pipelines.yml [hunk 1]
```diff
@@ -589,6 +589,32 @@ jobs:
...
+  - template: ci/download-artifacts-template.yml
+    parameters:
+      downloadPartialBuildBinFrom:
+        - chip_earlgrey_cw310
+        - sw_build
+  - bash: |
+      set -e
+      . util/build_consts.sh
+      module load "xilinx/vivado/$(VIVADO_VERSION)"
+      ci/scripts/run-fpga-tests.sh cw310 fpga_cw310_sival,-manuf,-hyper310 || { res=$?; echo "To reproduce failures locally, follow the instructions at https://opentitan.org/book/doc/getting_started/setup_fpga.html#reproducing-fpga-ci-failures-locally"; exit "${res}"; }
```
Comment (jwnrt) [inline azure-pipelines.yml:614]: Are you sure these should be running with `cw310` as the interface and not `hyper310`?

Comment (nbdd0121) [reply azure-pipelines.yml]: I think so but I am not 100% certain

Comment (a-will) [reply azure-pipelines.yml]: Yes, they should run on `cw310`. We haven't finished off `hyper310` support yet.

There are two related PRs with different approaches, and we need to sort out how we're going to handle tags:
1. https://github.com/lowRISC/opentitan/pull/20396
2. https://github.com/lowRISC/opentitan/pull/20390

In addition, I debugged some issues in hyperdebug firmware related to JTAG, and we'll need a fix to get in to get all the tests passing:
https://chromium-review.googlesource.com/c/chromiumos/platform/ec/+/5061310

**Review by engdoreis — APPROVED** [review]

**Pull request merged**